Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Im open minded. Which is best?
On 9 Feb 2015 04:29, "tim smith" <notifications@github.com> wrote:

> In Library/Formula/websockify.rb
> <https://github.com/Homebrew/homebrew/pull/36643#discussion_r24303590>:
>
> > @@ -0,0 +1,70 @@
> > +require "FileUtils"
> > +
> > +class Websockify < Formula
> > +  homepage "https://github.com/kanaka/websockify"
> > +  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
> > +  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
> > +
> > +  depends_on :python if MacOS.version <= :snow_leopard
>
> websockify has a numpy dependency. System python comes with numpy but a
> user-installed python may not.
>
> Do we want to make this
>
> if MacOS.version <= :snow_leopard
>   depends_on :python
>   depends_on "numpy" => :pythonend
>
> or do something more elaborate?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/36643/files#r24303590>.
>